### PR TITLE
added @snowpack/plugin-bundle-webpack

### PR DESCRIPTION
**Pull request by lajoiemedia**

### CODE REVIEW

File: packages/plugin-bundle-webpack/plugin.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                {
+                  loader: MiniCssExtractPlugin.loader,
+                },
+                {
+                  loader: "css-loader",
+                },
+              ],
+            },
+            {
+              test: /\.(svg|png|jpe?g|gif)$/, //We can expand this list of importable files
```
Comment (FredKSchott) [inline packages/plugin-bundle-webpack/plugin.js:78]: One nice thing about how Snowpack works is that the bundle stage only needs to worry about built files, so no sass, scss, jsx, tsx, etc. 

The way our dev server works, anything that is ESM imported that isn't JS or CSS is handled via something like file-loader, so I think we can just convert this test to `.*`

File: packages/plugin-bundle-webpack/plugin.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            filename: "css/style-[hash].css",
+          }),
+          //Copy other files to the destination, excluding ones that are no longer useful
+          new CopyPlugin({
+            patterns: [
+              {
+                from: srcDirectory,
+                to: destDirectory,
+                globOptions: {
+                  ignore: [`/${fallback}`, "**/_dist_/**", "**/web_modules/**"],
```
Comment (FredKSchott) [inline packages/plugin-bundle-webpack/plugin.js:103]: `_dist_` is CSA specific, and not general so we should avoid baking it into the plugin. But, I think this should be safe to remove anyway.

Comment (lajoiemedia) [reply packages/plugin-bundle-webpack/plugin.js:103]: But if we remove _dist_ from ignore, the _dist_ folder is going to get copied over from .build to build, isn't it?

File: packages/plugin-bundle-webpack/plugin.js (hunk 1 above)
Comment (lajoiemedia) [reply packages/plugin-bundle-webpack/plugin.js:78]: Yes that makes sense

File: packages/plugin-bundle-webpack/plugin.js (hunk 2 above)
Comment (FredKSchott) [reply packages/plugin-bundle-webpack/plugin.js:104]: hmm maybe, lets keep it in for now then

**Comment by FredKSchott** [comment]
A `webpack.extend.js` config file could be interesting. This PR is a great first step tho, I'll merge and then we can go from there

**Pull request merged**